bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -16,6 +22,16 @@
 
 public final class LaunchUtils {
 
+    private static final ILogger logger = Logger.getLogger(LaunchUtils.class);
+
+    private static final ServiceTracker<LaunchListener,LaunchListener> launchListeners;
+
+    static {
+        final BundleContext context = FrameworkUtil.getBundle(LaunchUtils.class).getBundleContext();
```

Comment:
Now that this is a static, is there a case where this class is loaded before the bundle it is part of is started? Then context would be null and the service tracker constructor will NPE resulting in this class failing to init. Seems like such a timing issue is possible, no?